app/pages/registration.vue

```diff
@@ -0,0 +1,108 @@
+<script setup>
+import { ref, computed, onMounted, watch } from 'vue';
+import intlTelInput from 'intl-tel-input';
+import useUserService from '~/composables/useUserService';
+import 'intl-tel-input/build/css/intlTelInput.css';
+import { form } from '#build/ui';
+
+const { registerUserMutation } = useUserService();
+
+const formData = ref({
+  firstName: '',
+  lastName: '',
+  email: '',
+  phone: '',
+});
+
+const phoneInput = ref(''); // Reference for the phone input
```

Comment:
```suggestion
const phoneInput = ref(null); // Reference for the phone input
```

Lets set this back to null